A3-channel-tracing.md

```diff
@@ -0,0 +1,111 @@
+Channel Tracing
+----
+* Author(s): Noah Eisen (ncteisen@google.com)
+* Approver: markdroth
+* Status: Draft
+* Implemented in: N/A
+* Last updated: 02/09/18
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/WFDj3KeHYTI
+
+## Table of Contents
+
+  * [Abstract](#abstract)
+  * [Background](#background)
+  * [Proposal](#proposal)
+  * [Format of Exported Data](#format-of-exported-data)
+
+## Terminology
+
+This design uses the same terminology discussed in the [channelz](A14-channelz.md) design. Channels and subchannels will be handled by the same channel tracing code, so for this design, a channel means either a channel or subchannel from the channelz terminology.
+
+In addition, the following terminology will be used:
+
+1.  a "Trace event" is an interesting thing that happens to a channel. Examples include things like creation, address resolution, subchannel creation, connectivity state changes.
+2. "Trace data" is all of the data that is contained for one channel. This includes a list of _Trace events_, as well as metadata like the timestamp at which the channel was created.
+3.  a "Tracing object" is an in-memory data structure responsible for holding the _Trace data_ for a single channel.
```

Comment:
If the only thing contained in the "Tracing object" is the "Trace data", then why is "Trace data" a separate object?  Why not simply put the fields from "Trace data" directly in the "Tracing object"?

Reply:
The tracing object also holds the child data. I will specify that

Reply:
I also removed the notion of trace data, because it was uneeded.